auth-api/src/auth_api/services/org.py

```diff
@@ -75,6 +75,7 @@ def create_org(org_info: dict, user_id, token_info: Dict = None):
         org = OrgModel.create_from_dict(camelback2snake(org_info))
         if is_staff_admin:
             org.access_type = AccessType.ANONYMOUS.value
+            org.billable = True
```

Comment:
This needs to be False

Reply:
oh..is the director search account non-bilable? ?I thought otherwise